### DISCUSSION TITLE
Possible to apply different color to the lower right ("estimated") part of the triangle?

### DISCUSSION THREAD

**Discussion by JorgenHorvei** [casact/chainladder-python#365] [Q&A]
Hi, 

I am writing a code that computes cumulative triangles and export them to excel. In this context, I wonder if it is possible to change the color of the lower right ("estimated") part of the triangle and export the triangle to excel keeping this new color formatting intact?

Best,
Jørgen

**Comment by jbogaardt** [comment]
This is out of scope for the `chainladder-python` library, but maybe I can provide general guidance.

- [openpyxl](https://openpyxl.readthedocs.io/en/stable/) allows for both read/write/modify of Excel files. It has an Excel VBA feel to it. I'm sure it supports formatting.
- [xlsxwriter](https://xlsxwriter.readthedocs.io/) is write only but is fast, definitely supports the formatting you're looking for, and can be used on different operating systems whether Excel is present or not.
- [xlcompose](https://github.com/jbogaardt/xlcompose) - a lightweight wrapper around `xlsxwriter` that just makes more complex Excel layouts and formats easier to accomplish. Since it wraps `xslxwriter`, it too is write only. I must disclose that I am the maintainer of this package.

There are others python to Excel packages and I am sure you can find them in your favorite search engine.
